lxc/list.go

```diff
@@ -367,8 +367,19 @@ func (c *listCmd) statusColumnData(cInfo shared.ContainerInfo, cState *shared.Co
 	return strings.ToUpper(cInfo.Status)
 }
 
+func cIsActive(cInfo shared.ContainerInfo) bool {
```

Comment:
Well, my point is that because all lxc/* commands share the same namespace, we're trying to avoid having global functions provided by some random command in there.

This one may actually make sense as a function against shared.ContainerInfo itself so that you can just do cInfo.IsActive()